inspirehep/factory.py

```diff
@@ -37,7 +41,7 @@ def config_loader(app, **kwargs_config):
 
 create_app = create_app_factory(
     "inspirehep",
-    config_loader=config_loader,
+    config_loader=api_config_loader,
```

Comment:
this is the wrong place, as this is the app factory. i was talking about the `create_api` factory above.

Reply:
When I added it to the first one it wasn't working.